classes/MusicAlbum/genre.rb

```diff
@@ -0,0 +1,20 @@
+class Genre
+  attr_reader :id, :name, :items
+
```

Comment:
it is better to add this line to your code to allow user to write and read. 
attr_accessor :name